### PR TITLE
combine bounce to mixins

### PR DESCRIPTION
**Pull request by hippogriffic**
blah

@incuna/frontend please review

### CODE REVIEW

File: hover/effects/background-transitions/_bounce-to.sass [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+//         @include bounce-to(top, black, white)
+//         background: white
+//         color: black
+
+// The following will produce a white button with black text and a 5px border-radius.
+//    On hover the text will transition from black to white. An ::after element with a black
+//    background will bounce across from left to right. The ::after will also have a 5px
+//    border-radius. The bounce will last 0.2s
+//     .button
+//         @include bounce(right, black, white, 5px, after, 0.2s)
```
Comment (pandalion) [inline hover/effects/background-transitions/_bounce-to.sass]: needs to be `include bounce-to` I think?

Comment (hippogriffic) [inline hover/effects/background-transitions/_bounce-to.sass]: yep, whoops, thanks

**Comment by hippogriffic** [comment]
@pandalion fixed

**Comment by ninjanails** [comment]
Looks good to me :+1: 

It looks so pretty

**Pull request merged**